mjpc/CMakeLists.txt

```diff
@@ -101,8 +101,6 @@ add_library(
   estimators/batch.cc
   estimators/batch.h
   estimators/estimator.h
-  estimators/gui.cc
```

Comment:
Looks like this is still used in batch.h, which is causing the build to fail.

Reply:
You are right. Looks like a merge issue with the parameters PR. I removed the gui.h include.